data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,110 @@
+          "enum": [
+            "foaf:Person",
+            "foaf:Organisation",
+            "oa:SoftwareAgent"
+          ]
+        },
+        "foaf:name": {
+          "type": "string",
+          "examples": ["My Name"],
+          "description": "The name of the creator. https://xmlns.com/foaf/0.1/#term_name."
+        }
+      },
+      "description": "The creator of the collection."
+    },
+    "ltc:ObjectGroup": {
+      "type": "object",
+      "properties": {
+        "ltc:conditionsOfAccess": {
+          "type": "string",
+          "examples": ["Open to the public"],
+          "description": " https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_conditionsOfAccess Information about who can access the collection being described or an indication of its security status."
+        },
+        "ltc:collectionName": {
+          "type": "string",
+          "examples": ["Arctiacaja1991Netherlands"],
+          "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_collectionName A short title that summarizes the collection objects contained within the ObjectGroup."
+        },
+        "ltc:description": {
+          "type": "string",
+          "examples": ["A virtual collection representing all butterfly specimens in South Holland"],
+          "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_description A free text description or narrative about the collection."
+        },
+        "ltc:isKnownToContainTypes": {
+          "type": "array",
+          "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_isKnownToContainTypes Flag property to indicate that the collection is known to include type specimens. `true` - this collection contains types; `false` - this collections does not contain types.",
+          "items": {
+            "type": "boolean",
+            "examples": ["true"],
+            "description": "Type known to be contained."
+          }
+        }
+      },
+      "description": "An intentionally grouped set of objects with one or more common characteristics."
+    },
+    "ltc:MeasurementOrFact": {
+      "type": "object",
+      "properties": {
+        "dwc:measurementType": {
+          "type": "string",
+          "examples": ["Object Count"],
+          "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_measurementType The nature of the measurement, fact, characteristic, or assertion."
+        },
+        "dwc:measurementValue": {
+          "type": "number",
+          "examples": [100],
+          "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_measurementValue The numerical value of the measurement."
+        }
+      },
+      "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_MeasurementOrFact A measurement of or fact about a class within the standard, or a relationship between the ObjectGroup and an associated class."
```

Comment:
Maybe put this description directly after the term, it feels a bit detached at the moment.